src/wanderer/guzzlr.ts

```diff
@@ -78,24 +79,31 @@ export function guzzlrFactory(
     if (location !== null) {
       const guzzlrBooze =
         Guzzlr.getTier() === "platinum" ? Guzzlr.getCheapestPlatinumCocktail() : Guzzlr.getBooze();
-      return [
-        new WandererTarget("Guzzlr", location, guzzlrValue(Guzzlr.getTier()), () => {
-          if (!guzzlrBooze) {
-            // this is an error state - accepted a guzzlr quest but mafia doesn't know the booze
-            return false;
-          }
+      return guzzlrBooze
+        ? [
+            new WandererTarget(
+              "Guzzlr",
+              location,
+              guzzlrValue(Guzzlr.getTier()) - mallPrice(guzzlrBooze),
```

Comment:
does garbo's mall price consider price of components vs price of the full one? i assume it does i am just curious